plutus-tx-plugin/src/PlutusTx/Compiler/Expr.hs

```diff
@@ -23,14 +23,13 @@ import PlutusTx.Compiler.Utils
 import PlutusTx.Coverage
 import PlutusTx.PIRTypes
 -- I feel like we shouldn't need this, we only need it to spot the special String type, which is annoying
-import PlutusTx.Builtins.Class qualified as Builtins
-import PlutusTx.Trace
-
 import Class qualified as GHC
 import CoreSyn qualified as GHC
 import CostCentre qualified as GHC
 import GhcPlugins qualified as GHC
 import MkId qualified as GHC
+import PlutusTx.Builtins.Class qualified as Builtins
+import PlutusTx.Trace
```

Comment:
Oh, that's a great example of how comments in the code get outdated as the code formatter is not smart enough to move them as well.